scripts/build/parse_syscalls.py

```diff
@@ -77,7 +77,11 @@ def analyze_headers(multiple_directories):
                     continue
 
                 with open(path, "r", encoding="utf-8") as fp:
-                    contents = fp.read()
+                    try:
+                        contents = fp.read()
+                    except Exception:
+                        sys.stderr.write("Error decoding %s\n" % path)
```

Comment:
you commit message states:
`that caused the `open` call to fail`
but this `try-except` is on the `read()`, not the `open()` ?

Generally, errors on `open()` will print the file is question, so could you please provide more details on the error / exception seen.

Reply:
Sorry, you're right, I wrote the commit message days after making the change and didn't write it correctly.